engine/lib/worker/exit_controller.go

```diff
@@ -71,6 +127,14 @@ func (c *ExitController) PollExit() error {
 		return nil
 	}
 
+	c.callPrepareOnce.Do(func() {
```

Comment:
Another problem concerns me a lot.
We design the `Tick` interface to implement the period behavior of business, but DM will run its logic in a independent goroutine, so we call `CloseImpl` before stop calling `Tick` seems ok.

But, to some extend,  other business will observe that `CloseImpl` will be called before stop calling `Tick`, which
means `a man is dead but its heart is beating`.

Reply:
It does not seem that `Tick` can be called after `CloseImpl`, based on the implementation of `BaseWorker`.

Reply:
My mistake.